sdk/opendp/whitenoise/client/restclient/models/dataset_document.py

```diff
@@ -27,14 +29,16 @@ class DatasetDocument(Model):
         'dataset_name': {'key': 'dataset_name', 'type': 'str'},
         'dataset_type': {'key': 'dataset_type', 'type': 'str'},
         'budget': {'key': 'budget', 'type': 'float'},
+        'released': {'key': 'released', 'type': 'bool'},
```

Comment:
I get that they are effectively the same. But it may be worth introducing a ReleaseDatasetDocument instead of having this flag. Overloading budget to budget used vs max budget seems problematic longer term. Especially given the very distinct semantics around both.

What do you think?

Reply:
I agree - can totally see DatasetDocument becoming bloated in the longtermwithout separating the two